Classes/Controller/FrontendLoginController.php

```diff
@@ -38,16 +37,23 @@ public function __construct(protected readonly ConfigurationService $configurati
      */
     public function loginAction(ServerRequestInterface $request): ResponseInterface
     {
-        $userUid = (int)$request->getQueryParams()['uid'];
+        $siteIdentifier = (string)$request->getQueryParams()['site'];
+        $userUid = (int)$request->getQueryParams()['user'];
 
         if ($userUid > 0) {
-            $pageUid = $this->configurationService->getRedirectPageId();
+            // redirect to site root should always be safe for login purposes -> "login redirect" happens in
+            // RedirectHandler middleware
+            $pageUid = GeneralUtility::makeInstance(SiteFinder::class)
+                                     ->getSiteByIdentifier($siteIdentifier)
```

Comment:
please don't align this according to the line before but only with 4 additional spaces

Reply:
Thank you for the tip, but I believe that indenting inline comments is a matter of personal preference. I don't know of any strict guidelines or best practices in PHP projects regarding this. Just search the TYPO3 core code using the following regex: `(.*)\/\/\s(.*)\n(.*)\/\/\s(.*)`
There are many more examples of such code comments.

I'm happy to receive suggestions for improving features, but the code is already automatically scanned by PHPStan and PHP CS Fixer (with TYPO3 Coding Guidelines) - that should be sufficient.

Reply:
I'm not talking about the comment but about the `->getSiteByIdentifier($siteIdentifier)`

Reply:
That's still personal preference, or simply a setting within one's IDE. It won't change anything regarding the functionality.